tokio/src/runtime/time/wheel/level.rs

```diff
@@ -120,7 +120,7 @@ impl Level {
     }
 
     pub(crate) unsafe fn add_entry(&mut self, item: TimerHandle) {
-        let slot = slot_for(item.registered_when(), self.level);
+        let slot = slot_for(unsafe { item.registered_when() }, self.level);
```

Comment:
Since almost all methods on `TimerHandle` is unsafe, and both of them share the same safety requirement as follow, it doesn't helpful to add additional "Safety" here.

https://github.com/tokio-rs/tokio/blob/5a709e391bdab3cf8bab9fe65d75fa2b92f4e0db/tokio/src/runtime/time/entry.rs#L315-L328